content/source/docs/cloud/users-teams-organizations/api-tokens.html.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: "cloud"
+page_title: "API Tokens - Terraform Cloud"
+sidebar_current: "docs-cloud-users-teams-organizations-api-tokens"
+---
+
+# API Tokens
+
+Terraform Cloud supports three distinct types of API tokens with varying levels of access: user, team, and organization. There are differences in access levels and generation workflows for each of these token types, which are outlined below.
+
+Like with [user tokens](./users.html#api-tokens), team and organization API tokens are displayed only once when they are created, and are obfuscated thereafter. If the token is lost, it must be regenerated. When a token is regenerated, the previous token immediately becomes invalid.
+
+~> **Important:** Unlike user tokens, team and organization tokens cannot authenticate with Terraform's `atlas` backend. This means they can't be used to run Terraform locally against state stored in Terraform Cloud, access `terraform_remote_state` data sources during a run, or [migrate local state into Terraform Cloud](../migrate/index.html). If you need to use the `atlas` backend for any reason, use a [user token][./users.html#api-tokens].
```

Comment:
I thought we finally got rid of this, that enough time has passed where the `atlas` backend is moot at this point.

Reply:
Oh you are right. I wrote the original commit so long ago it was actually relevant back in those days 😂 I'll just remove this.